css-two-column-layout/styles.css

```diff
@@ -0,0 +1,51 @@
+.container {
+  width: 70%;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.head-font-property {
+  font-weight: 700;
+  font-size: 40px;
+}
+
+.subhead-font-property {
+  margin-top: 40px;
+}
+
+.font-property {
+  font-family: 'Karla', sans-serif;
+}
+
+.column-one {
+  text-align: center;
+  width: 100%;
+}
+
+.column-two {
+  display: flex;
```

Comment:
If you feel tempted to put `display: flex` on a column class, that usually means that you should put a `row` class element in that column. Also, relocate the margin to another class as well